src/index.js

```diff
@@ -102,22 +102,24 @@ const container = document.getElementById('root')
 const root = createRoot(container)
 
 root.render(
-  <div style={{}}>
-    <div>
-      <h1>Demo using the &apos;react-oauth2-code-pkce&apos; package</h1>
-      <p>
-        Github:{' '}
-        <a href="https://github.com/soofstad/react-oauth2-pkce">https://github.com/soofstad/react-oauth2-pkce</a>
-      </p>
-      <p>
-        NPM:{' '}
-        <a href="https://www.npmjs.com/package/react-oauth2-code-pkce">
-          https://www.npmjs.com/package/react-oauth2-code-pkce
-        </a>
-      </p>
+  <React.StrictMode>
+    <div style={{}}>
```

Comment:
Not sure what the outermost div is doing anymore, I think it can be omitted.

Reply:
Fixed!